### PR TITLE
Add flash.display.StageAspectRatio

### PR DESCRIPTION
**Pull request by Fancy2209**
also stubs stage.setAspectRatio
Currently a draft because I somehow made it impossible to load SWF

### CODE REVIEW

**Comment by Fancy2209** [comment]
Somehow SWF can't load anymore, what did I do 😓

**Comment by Fancy2209** [comment]
It seems API Versioning is what broke ruffle?????

File: core/src/avm2/globals/flash/display/StageAspectRatio.as [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+{
+    [API("668")]
+    public final class StageAspectRatio
+    {
+        [API("681")]
+        public static const ANY:String = "any"
+        [API("668")]
+        public static const LANDSCAPE:String = "landscape"
+        [API("668")]
+        public static const PORTRAIT:String = "portrait"
```
Comment (kjarosh) [inline core/src/avm2/globals/flash/display/StageAspectRatio.as:11]: You are missing semicolons, hence the errors. Also, I think annotating these constants as `[API("668")]` is unnecessary, as the whole class is annotated that way.

Comment (Fancy2209) [reply core/src/avm2/globals/flash/display/StageAspectRatio.as]: Fixed

**Review by kjarosh — APPROVED** [review]

**Comment by torokati44** [comment]
Thank you for cleaning up the commit history! :pray:

**Pull request merged**